atom/browser/node_debugger.cc

```diff
@@ -37,15 +37,14 @@ void NodeDebugger::Start() {
     // Use custom platform since the gin platform does not work correctly
     // with node's inspector agent
     platform_.reset(new node::NodePlatform(
-        /* thread_pool_size */ 0,
-        env_->event_loop(),
+        /* v8_default_thread_pool_size */ 4, env_->event_loop(),
```

Comment:
`thread_pool_size ` was a name of an argument there. This kind of comments is quite common in the Chromium code, because if you don't remember exactly the signature of a function, it's quite difficult to guess what `0` might mean here.

Reply:
Yeah I should change it back. My initial intention was to specify the reason for choosing 4 along the same line, since node.cc declares a `v8_default_thread_pool_size`. But can be documented separately and not mess with argument comments.

Reply:
Great, thanks!